app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
u could unpack it like this 
var1,var2,var3 = command.split()
to avoid using indexes